jmh-core/src/main/java/org/openjdk/jmh/util/AbstractStatistics.java

```diff
@@ -93,7 +93,7 @@ public int compareTo(Statistics other, double confidence) {
         if (isDifferent(other, confidence)) {
             double t = getMean();
             double o = other.getMean();
-            return (t > o) ? -1 : 1;
+            return (t < o) ? -1 : 1;
```

Comment:
Should it be `return Double.compareTo(t, o)`?

Reply:
This is a good suggestion, done so.